app/components/inputs/outlined-input.jsx

```
@@ -0,0 +1,30 @@
+import React from 'react';
+import '@material/web/textfield/outlined-text-field.js';
+import PropTypes from 'prop-types';
+import { type } from 'os';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)